src/Core/Checkout/Cart/Event/BeforeLineItemQuantityChangedEvent.php

```diff
@@ -28,10 +33,12 @@ class BeforeLineItemQuantityChangedEvent implements ShopwareSalesChannelEvent, C
     protected $salesChannelContext;
 
     public function __construct(
+        int $beforeUpdateQuantity,
```

Comment:
this is a breaking change, maybe add a separate setter for that or directly use a public property?